src/main/resources/db/migration/V2__create_catigories.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE categories (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
```

Comment:
you probably want to remove the final column here on line 3 -- was this running?